roles/velero/defaults/main.yml

```diff
@@ -12,9 +12,18 @@ velero_state: "present"
 velero_aws_bsl_configs: 0
 velero_gcp_bsl_configs: 0
 velero_azure_bsl_configs: 0
+velero_aws_vsl_configs: 0
+velero_gcp_vsl_configs: 0
+velero_azure_vsl_configs: 0
 velero_aws_secret_name: cloud-credentials
 velero_gcp_secret_name: gcp-cloud-credentials
 velero_azure_secret_name: azure-cloud-credentials
+velero_aws_bsl: false
```

Comment:
Do we need to define these vars though? It feels to me like we can just wait for `set_fact` to set them. There's no world where the user sets these themselves right?